android/app/build.gradle

```diff
@@ -180,19 +170,11 @@ android {
     }
     sourceSets {
         // TODO: refactor making sure notifications are working properly both on debug and release
-        experimentalPlayDebug {
-            java.srcDirs = ['src/main/java', 'src/play/java']
-            manifest.srcFile 'src/play/AndroidManifest.xml'
-        }
-        experimentalPlayRelease {
-            java.srcDirs = ['src/main/java', 'src/play/java']
-            manifest.srcFile 'src/play/AndroidManifest.xml'
-        }
-        officialPlayDebug {
+        playDebug {
```

Comment:
Wouldn't it be just debug and release, instead of PlayDebug?
What does Play mean in these cases?

Reply:
We still have Foss version, so it's either Foss or Play for every task.